conversions/vrep_conversion.cpp

```diff
@@ -0,0 +1,82 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <cstring>
+#include <mrpt/obs/CObservation2DRangeScan.h>
+#include "mrpt_bridge/vrep_conversion.h"
+#include <mrpt/math/CQuaternion.h>
+#include <mrpt/obs/CObservationOdometry.h>
+
+using namespace mrpt::obs;
+using namespace mrpt::poses;
+using namespace mrpt::math;
+
+namespace vrep_bridge
+{   /**
+    *This method converts VREP laser scans into a CObservation2DRangeScan class object. It has the following arguments:
+    *- <b>_range[]</b> -> This array contains all the laser scan measurements(distances).
```

Comment:
No need for bold there..

Use `@param [in]` or `@param[out]` here instead

https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdparam